apps/web/src/app/[lang]/network/page.tsx

```
@@ -3,12 +3,24 @@ import { Container } from '@hypha-platform/ui';
-import { createSpaceService } from '@hypha-platform/core/server';
+import { createSpaceService, Space } from '@hypha-platform/core/server';
+import { Category } from '@hypha-platform/core/client';
 
 type PageProps = {
   params: Promise<{ lang: Locale; id: string }>;
 };
 
+function extractUniqueCategories(spaces: Space[]) {
+  const categories = spaces.reduce<Category[]>((acc, space) => {
+    if (space.categories) {
+      return [...acc, ...space.categories];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)